src/append/mod.rs

```diff
@@ -12,10 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Dispatch log records to the appropriate target.
+//! Various appenders for log records.
```

Comment:
```suggestion
//! Dispatch log records to the appropriate target.
```

Keep the original doc. It's more appropriate to me.